Bottom: a61da5dd4fb8e9001ac399ab473a13ecd527d81b
Top:    98f3cf3eb8418e00367fa8de9c2cc4605d6e45fd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 02:12:27 +0100

hid/gtk: Ask for polygons after masking as well as before

The GL renderer will need both, so add both now. Teach the GDK renderer
to the ignore gui->use_mask (HID_MASK_AFTER) calls it will now get.


---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 28c10bc..4f16d8c 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -979,7 +979,7 @@ HID ghid_hid = {
   1,				/* gui */
   0,				/* printer */
   0,				/* exporter */
-  0,				/* poly before */
+  1,				/* poly before */
   1,				/* poly after */
   0,				/* poly dicer */
